tests/conftest.py

```diff
@@ -51,28 +53,40 @@
     collect_ignore_glob.append("*test_couchbase*")
 
 if not os.environ.get("GEVENT_STARLETTE_TEST"):
-    collect_ignore_glob.extend([
-        "*test_gevent*",
-        "*test_starlette*",
-    ])
+    collect_ignore_glob.extend(
+        [
+            "*test_gevent*",
+            "*test_starlette*",
+        ]
+    )
 
 if not os.environ.get("KAFKA_TEST"):
     collect_ignore_glob.append("*kafka/test*")
 
+# Currently asyncio and tornado_server depends on aiohttp and
+# since aiohttp versions < 3.12.14 have vulnerability we skip the tests below
+if sys.version_info < (3, 9):
+    collect_ignore_glob.append("*test_aiohttp*")
+    collect_ignore_glob.append("*test_asyncio*")
+    collect_ignore_glob.append("*test_tornado_server*")
+
```

Comment:
Please start using the `collect_ignore_glob.extend([])` as in the other blocks.

```suggestion
if sys.version_info < (3, 9):
    collect_ignore_glob.extend([
        "*test_aiohttp*",
        "*test_asyncio*",
        "*test_tornado_server*",
    ])

```